build.sbt

```diff
@@ -293,9 +293,9 @@ lazy val scroogeSbtPlugin = Project(
 ).settings(
   // we remove the publishTo key because bintray already sets it, and setting
   // it again breaks the bintray plugin.
-  settingsWithTwoTen.filter(_.key.key.label != "publishTo")
+  sharedSettingsWithoutScalaVersion.filter(_.key.key.label != "publishTo")
```

Comment:
since we're changing this now, I don't think we ever use `settingsWithTwoTen` anymore.  can you delete that val?

Reply:
I removed this val in 1c690c2.